stdlib/collections/__init__.pyi

```diff
@@ -96,6 +96,11 @@ class UserDict(MutableMapping[_KT, _VT], Generic[_KT, _VT]):
         def __ior__(self, other: SupportsKeysAndGetItem[_KT, _VT]) -> Self: ...
         @overload
         def __ior__(self, other: Iterable[tuple[_KT, _VT]]) -> Self: ...
+    if sys.version_info >= (3, 12):
+        @overload
+        def get(self, key: _KT, default: None = None) -> _VT | None: ...
+        @overload
+        def get(self, key: _KT, default: _VT | _T) -> _VT | _T: ...
```

Comment:
Isn't the `_VT` in the default parameter redundant and something that was previously causing bugs? I seem to recall some issue and PR regarding that, because when the default is of `_VT`, then `_T` remains unbound and results in an `Any` in the return value union.

Reply:
Hmmmmm. Yes, I think you're right. Let me play around a little.